package.json

```
@@ -3,11 +3,13 @@
   "version": "1.11.4",
   "description": "An immutable duration library based on the ISO-8601 format for durations.",
   "main": "lib/index.js",
+  "module": "lib/index.esm.js",
   "readme": "README.md",
   "sideEffects": false,
   "scripts": {
     "rebuild-docs-index": "node src/_generate_docs.js > README.md && node src/_generate_index.js > src/index.js && git commit -m 'Regenerate docs and index' README.md src/index.js || true",
-    "build": "npm run rebuild-docs-index && babel src/ --out-dir lib/ --ignore *.spec.js",
+    "build": "npm run rebuild-docs-index && babel src/ --out-dir lib/ --ignore *.spec.js && npm run build esm",
```

Otherwise looks good

```suggestion
    "build": "npm run rebuild-docs-index && babel src/ --out-dir lib/ --ignore *.spec.js && npm run build:esm",
```